### PR TITLE
add CULL_MESHOPT constant

### PR DESCRIPTION
**Pull request by PPpro**
Re: #

### Changelog

* add CULL_MESHOPT constant

### Rely on PR
https://github.com/cocos/cocos-editor/pull/2518

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by yiwenxue — APPROVED** [review]

File: cc.config.json [hunk 1]
```diff
@@ -678,6 +678,12 @@
             "type": "boolean",
             "value": false,
             "internal": true
+        },
+        "CULL_MESHOPT": {
+            "comment": "An internal constant to indicate whether we cull the meshopt wasm module and asm.js module.",
+            "type": "boolean",
+            "value": false,
```
Comment (PPpro) [inline cc.config.json]:
```suggestion
            "value": true,
```

File: cc.config.json [hunk 2]
```diff
@@ -678,6 +678,12 @@
             "type": "boolean",
             "value": false,
             "internal": true
+        },
+        "CULL_MESHOPT": {
```
Comment (minggo) [inline cc.config.json:682]: Where to set the value?

Comment (PPpro) [reply cc.config.json:682]: in editor 
https://github.com/cocos/cocos-editor/pull/2518

Comment (minggo) [reply cc.config.json:682]: In what situation needs to cull meshopt? And what situation doesn't need to cull?

**Review by minggo — APPROVED** [review]

File: cc.config.json (hunk 2 above)
Comment (PPpro) [reply cc.config.json:682]: cull meshopt in order to reduce the engine package size.
but with meshopt, we can reduce mesh assets size.

**Pull request merged**